python-avd/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Since we vendor ansible-avd which is Apache-2.0, we should also be doing Apache2.0 here.

Reply:
Fixed!